scripts/rook.sh

```diff
@@ -84,29 +83,19 @@ kubectl_retry() {
 function deploy_rook() {
 	kubectl_retry create -f "${ROOK_URL}/common.yaml"
 	kubectl_retry create -f "${ROOK_URL}/operator.yaml"
-	# find out the rook version to decide on the ceph cluster image to be used
-	ROOK_MAJOR=$(rook_version 1)
-	ROOK_MINOR=$(rook_version 2)
-	if { [ "${ROOK_MAJOR}" -eq 1 ] && [ "${ROOK_MINOR}" -le 2 ]; }; then
-		ROOK_CEPH_CLUSTER_VERSION_IMAGE_PATH="image: ceph/ceph:${ROOK_CEPH_CLUSTER_VERSION}"
-		# upgrade ceph cluster version to 14.2.10 to support CephFS snapshot functionalities.
-		TEMP_DIR="$(mktemp -d)"
-		curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
-		sed -i "s|image.*|${ROOK_CEPH_CLUSTER_VERSION_IMAGE_PATH}|g" "${TEMP_DIR}"/cluster-test.yaml
-		cat  "${TEMP_DIR}"/cluster-test.yaml
-		kubectl_retry create -f "${TEMP_DIR}/cluster-test.yaml"
-		rm -rf "${TEMP_DIR}"
-	else
-		# add "mon_warn_on_pool_no_redundancy = false" to ceph.conf if missing
-		# see https://github.com/rook/rook/pull/5925 for upstream status
-		TEMP_DIR="$(mktemp -d)"
-		curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
-		if ! grep -q mon_warn_on_pool_no_redundancy "${TEMP_DIR}"/cluster-test.yaml; then
-			sed -i '/osd_pool_default_size =/a \    mon_warn_on_pool_no_redundancy = false' "${TEMP_DIR}"/cluster-test.yaml
-		fi
-		kubectl_retry create -f "${TEMP_DIR}/cluster-test.yaml"
-		rm -rf "${TEMP_DIR}"
-	fi
+    # Override the ceph version which rook installs by default.
```

Comment:
I really wonder what happens with the indenting here on every update... Now the indention is different with the lines above?!

Reply:
It looks fine in my editor, let me checkout the branch somewhere else and recheck.

Reply:
might be tabs vs spaces?

Reply:
Looks like Still indentation needed to be fixed. Can you please check it

Reply:
I have checked many times, even reset everything using only tabs but it looks fine on my system.
Even git diff doesn't show anything after reset, I don't know why it is showing different here :(

Reply:
I suggest to merge it if there are no further comments and if there is still a problem I can raise a new PR.
Let me know what you guys think.